scripts/fit_image.py

```diff
@@ -0,0 +1,117 @@
+"""
+A python tool to resize images 
+
+Usage:
+    python fit_image.py --images (image_dir|one image) --width 1920
+
+    if you wanna resize image by setting resolution
+    please use this command :
+        - python fit_image.py --images xxxx --width 1920 (recommend)
+        - python fit_image.py --images xxxx  --height 1080 
+        - python fit_image.py --images xxxx --width 1920 --height 1080  
+
+    if you wanna resize image resolution by setting ratio ,
+    please use this command(not recommend):
+        - python fit_image.py --images xxx --ratio 0.5
+
+"""
+
+import os
+from PIL import Image
+import argparse
+import re
+import sys
+
+
+def chose_proper_resolution(args, old_width, old_height):
+    """
+    according the ratio or weight and hegiht to chose a proper resolution
+
+    return the proper resolution
+
+    """
+
+    if args.ratio > 0:
+        """
+        use ratio to resize (0,1) .
+        It's very dangerous action which will modify all image ,
+        so you need to make sure you wamma do this
+        """
+        if args.ratio > 1:
+            print("make sure ratio between 0 and 1")
+            exit()
+
+        print('-->Please check use ratio {} to resize all images!\
+            \n If you sure,please input yes/y to argee'.format(args.ratio))
+        input_str = sys.stdin.readline().strip().lower()
+        if 'yes' == input_str or 'y' == input_str:
+            target_width = int(old_width * args.ratio)
+            target_height = int(old_height * args.ratio)
+        else:
+            print('\n-->You need to think about this operation')
+            exit(1)
+    else:
+        # use width or height to resize
+        if old_width > args.width > 0 and old_height > args.height > 0:
+            # when width and height both are set
+            target_width = args.width
+            target_height = args.height
+        elif old_width > args.width > 0 and args.height == 0:
+            # when only width is set
+            target_width = args.width
+            target_height = int(args.width / old_width * old_height)
+        elif args.width and old_height > args.height > 0:
+            # when olny height is set
+            target_width = int(args.height / old_height * old_width)
+            target_height = args.height
+        else:
+            # other cases, don't modify images
+            target_width = old_width
+            target_height = old_height
+
+    return target_width, target_height
+
+
+def resize_image(image_path, args):
+    """
+    According ratio or resolution to resize image and relpace source image
+    """
+    img = Image.open(image_path)
+    old_width, old_height = img.size
+    target_width, target_height = chose_proper_resolution(
+        args, old_width, old_height)
+    if old_height == target_height and old_width == target_width:

```

Comment:
I think here we should better to use `<=` instead of `==` because `<=` will be more clear for our process logic.

Reply:
OK, this will indeed get better. fix it